catalog/admin.py

```diff
@@ -1,3 +1,29 @@
 from django.contrib import admin
-
+from .models import Book,Author,Language,BookInstance,Genre
 # Register your models here.
+# admin.site.register(Book)
+class BooksInstanceInline(admin.TabularInline):
+    model = BookInstance
+@admin.register(Book)
+class BookAdmin(admin.ModelAdmin):
```

Comment:
cách giữa 2 block 1 line trống nha em
```suggestion
@admin.register(Book)

class BookAdmin(admin.ModelAdmin):
```